### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_Add External Streaming Job_Keyboard: Expand collapse button present in 7th line in code editor is not accessible with any keyboard key

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#14359]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG2.1.1;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.27.0-insider
Windows Version: Windows10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab and right click on any created project.
5. Tab till Add External Streaming Job and hit enter
6. Tab to 7th line in code editor and try to access expand collapse button with keyboard keys

### Actual:
Expand collapse button present in 7th line in code editor is not accessible with any keyboard key.

Similar issue is present throughout the application where "Expand collapse button present" button is present in editor.

### Expected:
Expand collapse button should be accessible with the help of keyboard.

### User Impact:
Keyboard user will not be able to expand or collapse the code.

**Comment by alanrenmsft** [comment]
there are shortcut keys defined for fold/unfold actions,
1. go to command palette (main menu-> View -> Command Palette...)
2. run command 'Open Keyboard Shortcuts'
3. look for editor.fold and editor.unfold

![image](https://user-images.githubusercontent.com/13777222/112365624-1aa57980-8c95-11eb-80e0-e80ffa02c31b.png)

![image](https://user-images.githubusercontent.com/13777222/112365665-2729d200-8c95-11eb-9ad1-9b19b15a85a6.png)

**Comment by Swati1700** [comment]
Verified the bug in the environment
Application Name: Azure Data Studio
macOS: Ventura 13.2
Issue is fixed and does not reproduced. Please find the not repro attachment below.
[Screen Recording (7).webm](https://user-images.githubusercontent.com/86710888/220588221-258a6b56-4686-4567-9335-014bb6a8e220.webm)
Github Tags:#Closed;
